docs/en/toolkit/toolkit_basics_guides/advanced_config.md

```diff
@@ -155,13 +155,13 @@ And now the real fun begins, learning all the things you can do with the Configu
+Shotgun Toolkit provides many convenient ways to edit, clone, or take over a configuration. Extending existing configurations will save time and allow you access to all of the cool stuff that others within your network have created. You can take advantage of the vast Shotgun community (https://groups.google.com/a/shotgunsoftware.com/forum/?fromgroups#!forum/shotgun-dev) that may have the exact configuration you need. The Shotgun community is a sharing community, so be kind, say thank you, and recognize the person who created the configuration that helped you get the job done. Oh, and don’t forget to give back, it’s how we help our fellow Shotgun gurus and what makes it so special to be a part of this community! 
 
 Below are some ways you can have fun with configurations.
 
 ## Using the command line to create a default configuration
 
-From inside any project configuration, the `tank` command lets you run administrative commands from a terminal. Each project has its own dedicated `tank` command. The `tank setup_project` command's functionality is analogous to the Advanced Setup Wizard's: it creates an editable configuration on disk for your project based either on an existing project's configuration or the Default Configuration. You can [learn more about running `tank setup_project` here](https://support.shotgunsoftware.com/hc/en-us/articles/219033178-Administering-Toolkit#setup_project), and [more about the  `tank` command here](https://support.shotgunsoftware.com/hc/en-us/articles/219033178-Administering-Toolkit#Using%20the%20tank%20command).
+From inside any project configuration, the `tank` command lets you run administrative commands from a terminal. Each project has its own dedicated `tank` command. The `tank setup_project` command's functionality is analogous to the Advanced Setup Wizard's: it creates an editable configuration on disk for your project based either on an existing project's configuration or the Default Configuration. You can learn more about running [`tank setup_project` here](https://support.shotgunsoftware.com/hc/en-us/articles/219033178-Administering-Toolkit#setup_project), and more about the  [`tank` command here](https://support.shotgunsoftware.com/hc/en-us/articles/219033178-Administering-Toolkit#Using%20the%20tank%20command).
 
```

Comment:
I'm not sure why you made this change  – can you clarify?

Reply:
I noticed you added the cloud distribution section and was wondering if we want to talk a little about the benefits of a distributed configuration? Like the users always have the most up to date config since it's cached every time... I tried to imply this. Let me know what you think.
This was in the PR comment ... For the links in the advanced topics section... The community link is an external link, so it's visible in the text. The style guide states to use doc name or a term reference in the support guide as a link. See Style Guide... https://wiki.autodesk.com/pages/viewpage.action?spaceKey=SHOT&title=Style+Guide#StyleGuide-Linkingtothesupportsite  I left the tank links in, but moved the link to only the words "here"...
These should change too and remove the word here making the link something else... I didn't know what to change it to and wanted to leave that one to you.